docs/src/components/PlaygroundExample/index.js

```diff
@@ -2,14 +2,42 @@ import Link from "@docusaurus/Link";
 import { MDXProvider } from "@mdx-js/react";
 import CodeBlock from "@theme/CodeBlock";
 import * as components from "@theme/MDXComponents"; // Import default MDX components from Docusaurus theme
-import React from "react";
+import React, { useState } from "react";
 
 import RunSnippet from "../RunSnippet";
 import SideBySideColumn from "../SideBySide/Column";
 import SideBySideContainer from "../SideBySide/Container";
+import styles from "./styles.module.css";
+
+function getTitle(titleSize, title) {
```

Comment:
Should this at least do some min/max bound checking. E.g. if I put in 100, I still only get h6?